ravenml/data/helpers.py

```diff
@@ -0,0 +1,311 @@
+import os
+import shutil
+import pandas as pd
+import json
+import sys
+from random import shuffle
+from queue import Queue
+from threading import Thread
+from pathlib import Path
+from colorama import Fore
+from ravenml.utils.question import cli_spinner, user_selects, user_confirms, user_input
+from ravenml.utils.config import get_config
+
+def default_filter(tags_df, filter_metadata):
+    """Method leads user through interactive filtering through image_ids based on 
```

Comment:
Additional parameters (filtering by the amount of images needed as well as by tags) are being worked on, will most likely be in a separate pr.

Reply:
I thought tag filtering and selecting only a subset of images were already part of the old Jigsaw, are those just being set up for new structure?

Reply:
The way the jigsaw filtering worked was that you can filter by certain tags (AND/OR between them) and then select a subset of those images. The feature I was referring to is the ability to just choose the number of images you want from each of your imagesets without having to deal with tags. The feature is finished and in the branch `better-filtering` which I will make a PR for after this one goes through.